### PR TITLE
Filters and sorting added 

### PR DESCRIPTION
**Pull request by msdeshmukh009**
### Added filters on the product listing page.

**Sorting Option**
- Price-low to high
- Price-high to low

**Filter options**
- Filter by category: Coffee mugs, Posters, T-shirts, Accessories.
- Filter by rating
- Include out of stock

### CODE REVIEW

**Review by manojasarsa — APPROVED** [review]
The code looks consistent and clean.
I liked your range slider with two handles.
Good to go 👍

File: src/components/navbar/navbar.css [hunk 1]
```diff
@@ -2,6 +2,9 @@
 .nav {
   box-shadow: var(--nav-bar-shadow);
   grid-area: nav;
+  z-index: 1005;
+  max-width: 1800px;
+  margin: auto;
 }
 .nav-header {
   width: 200px;
```
Comment (pritam-kr) [inline src/components/navbar/navbar.css]: you can use relative units.

File: src/components/vertical-card/VerticalCard.jsx [hunk 2]
```diff
@@ -1,12 +1,34 @@
...
         <i className="fas fa-heart-circle"></i>
       </button>
       <div className="card-image-container">
-        <img className="responsive-img rounded-top-corner-img" src={image} alt="card-img" />
+        <img
+          className="responsive-img rounded-top-corner-img"
+          src={image}
+          alt={description}
+          width="340px"
+          height="270px"
```
Comment (pritam-kr) [inline src/components/vertical-card/VerticalCard.jsx]: remove px from it

File: src/pages/product-listing/FilterList.jsx [hunk 3]
```diff
@@ -11,71 +17,103 @@ const FilterList = ({ filterStyles, setFilterStyles }) => {
...
         <li className="filter-list-item">
           <h3>Price</h3>
           <div>
-            <input type="radio" name="price" id="high-to-low" />
+            <input
+              type="radio"
+              name="price"
+              id="high-to-low"
+              checked={state.sort === "high-to-low"}
+              onChange={() => dispatch({ type: "SORT", payload: "high-to-low" })}
```
Comment (pritam-kr) [inline src/pages/product-listing/FilterList.jsx:39]: you can use const variable instead of string

**Review by pritam-kr — APPROVED** [review]
Everything looks good to me.

**Pull request merged**